pkg/errors/errors.go

```diff
@@ -0,0 +1,12 @@
+package errors
+
+import "github.com/giantswarm/microerror"
+
+var NotFound = &microerror.Error{
+	Kind: "NotFoundError",
+}
+
+// IsNotFound asserts NotFound.
+func IsNotFound(err error) bool {
```

Comment:
Not sure how useful this matcher is compared to `errors.Is()` but I added it since it's relatively idiomatic still, especially in kubernetes related code.